### PR TITLE
Test documentation code blocks

### PR DESCRIPTION
**Pull request by RemDelaporteMathurin**
## Proposed changes

This is closing #718 by making use of `doctest` to test code blocks in the documentation.
That will make the docs more reliable in the future as changes will be found immediately!
Plus it helped fixing existing issues in the documentation.....

## Types of changes

What types of changes does your code introduce to FESTIM?

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Code refactoring
- [x] Documentation Update (if none of the other choices apply)
- [ ] New tests

### CODE REVIEW

File: docs/source/userguide/export_post_processing.rst [hunk 1]
```diff
@@ -140,49 +146,55 @@ Finally, you can add the :class:`festim.DerivedQuantities` object to the simulat
...
     )
 
-    my_model.exports = [my_derived_quantities, ....]
+    my_model.exports = [my_derived_quantities]
 
 
 The complete list of derived quantities can be found at: :ref:`Exports`.
 
 The data can be accessed in three different ways:
 - directly from the :class:`festim.DerivedQuantities` (plural) object:
```
Comment (KulaginVladimir) [inline docs/source/userguide/export_post_processing.rst:155]: The PR looks fine to me. However, this part lacks an example on the [filter](https://festim.readthedocs.io/en/latest/api/exports.html#festim.DerivedQuantities.filter) method, isn't it? Probably, it might be added in a dedicated PR.

Comment (RemDelaporteMathurin) [reply docs/source/userguide/export_post_processing.rst:155]: Agreed! I'll open an issue
